docs/ww3_doxy_tmpl.md

```diff
@@ -0,0 +1,68 @@
+# WAVEWATCH III Doxygen Header Reference
+<br>
+
+_Doxygen markup headers to be placed directly above the respective Fortran code units._
+
+### FILE
+```
+!> @file  <file name>
+!> @brief  <one sentence description>
+!> 
+!> @author <author name> @date <dd-Mon-yyyy>
+```
+
+### MODULE
+```
+!> @brief <one sentence description>   # contents of _1. Purpose_ from original header
+!> 
+!> @details <extended description>
+!>
+!> @param <module var 1>               # ALL module variables documented
+!> ...
+!> @param <module var N>
+!> @author <author name> @date <dd-Mon-yyyy>
+```
+
+### PROGRAM
+```
+!> @brief  <one sentence description>      # contents of _1. Purpose_ from original header
+!>
+!> @details <extended description>          # contents of _2. Method_ from original header
+!> 
+!> @author <author name>  @date <dd-Mon-yyyy>
```

Comment:
Actually the date field can be specified in any format. It's not ever really parsed as a date, it's just treated as text. So you could have:

`@date Sometime Last Winter
`
and it would work.

Sometimes in NCEPLIBS all we know is the year. ;-)

Reply:
I agree, though the particular format chosen was what has been used in the existing WW3 code headers, so was meant to be consistent from that standpoint.

Reply:
This was left as is for consistency across the current WW3 code base.